build/charts/antrea/conf/antrea-agent-secondary-network.conf

```diff
@@ -0,0 +1,37 @@
+# Name of the primary OpenVSwitch bridge antrea-agent will create for Secondary interface needs.
+# Make sure it doesn't conflict with your existing OpenVSwitch bridges.
+# Defaults to br-int-secnet1.
```

Comment:
Could we remove "1" and use just "br-int-secnet"?

Reply:
Yes, got it aligned.